content/en/core/process/workflow.md

```diff
@@ -140,9 +140,56 @@ Regardless of what you do with the issue, please:
+# Documentation Workflow
 
-Where possible, please follow our [docs style guide]({{< ref "docs-style-guide" >}}).
+This documentation site (and related [GitHub site](https://github.com/medic/medic.github.io)) is the CHT’s main resource for informing users on the product. It is designed to provide Community Health Toolkit users clear information on product functionality, as well as the necessary guidelines to accomplish key steps for setting up, deploying, and maintaining the product. 
 
+Documentation is a collaborative effort among all Community members. To improve information on the documentation site, we encourage commenting, asking questions, and sharing ideas for improvement.
 
+## Types of Documentation
+
+Documentation generally falls into three categories:
+
+### Feature Education
+Mainly prepared for a non-technical or semi-technical audience, it focuses on the [core features of the CHT]({{< ref "apps/features" >}}). Feature education includes summaries of feature functionality and screenshots with annotations or explanations of specific parts of the workflow as needed for added clarity. 
+
+### Technical Documentation
+Mainly prepared for a technical audience, it describes the [CHT Core Framework]({{< ref "core" >}}) and its [reference components]({{< ref "apps/reference" >}}). It includes information on technical implementation, design decisions, architecture descriptions, schemas, and source code. 
+
+### How-to Guides and Tutorials
+Mainly prepared for implementing partner capacity building, it includes best practices [quick guides]({{< ref "apps/guides" >}}) and [tutorials]({{< ref "apps/tutorials" >}}) on functional implementation of a feature (or group of features) and system configuration to assist users in completing specific objectives. 
+
+## Issues
+
+All documentation and technical documentation site issues are managed in [Github cht-docs](https://github.com/medic/cht-docs/issues). When creating issues, add the appropriate [labels](https://github.com/medic/cht-docs/labels) to help provide guidance to other community members.
+
+## Prioritization
+
+CHT documentation is treated similar to CHT code: the aim is to improve it as often as possible. Documentation is prioritized in three different ways:
+
+1. Feature education is completed at the time of releasing the related issue in GitHub. At the beginning of each release, an issue(s) is created, it is added to the release project, and a lead is assigned. 
+
+2. Technical documentation is completed at the time of closing the related issue in a GitHub release. 
+
+3. How-to guides and tutorials are prioritized alongside other product work as part of the roadmap planning process.
+
```

Comment:
I don't think this section is necessary, as the requirements for documentation should be listed in the dev workflow. You could include a not about when docs are introduced in the doc structure overview above. For example: "Reference documentation is updated as part of GH issues in CHT Core", "New guides are developed as part of the release materials when a new feature is introduced".

Reply:
Good point. I deleted this section